Volume/Supporting/Notifications.swift

```diff
@@ -67,6 +70,10 @@ class Notifications: NSObject {
         guard let url = URL(string: "\(Secrets.openArticleUrl)\(id)") else { return }
         UIApplication.shared.open(url)
     }
+    
+    private func openWeeklyDebrief() {
+        openedWeeklyDebrief = true
```

Comment:
`isWeeklyDebriefOpen`? Opened weekly debrief is very imperative language. With SwiftUI declarative language is better